source/common/network/io_socket_error_impl.h

```diff
@@ -31,6 +33,9 @@ class IoSocketError : public Api::IoError {
 
 private:
   int errno_;
+  Api::IoError::IoErrorCode error_code_;
```

Comment:
I am curious why `error_code_` is not const but can be used by const method `getErrorCode()`

Reply:
I actually made both const in febdfbc88c7bdf0f2ef1e6baf4156d5e89610bb1